.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Why only run these on these pull request events? We normally run them on all PR events by using

```
on:
  pull_request: {}
```

Reply:
I cribbed from the SDK: https://github.com/meltano/sdk/blob/main/.github/workflows/test.yml#L5 but switching to all PR events definitely makes sense. Will update 👍